packages/connector-thegraph/src/voting/entities/Voting.ts

```diff
@@ -0,0 +1,18 @@
+import VotingEntity from "./VotingEntity";
+import Vote from "./Vote";
+import VotingConnectorTheGraph from "..";
+import { App } from "plumbery-core";
+
+export default class Voting extends VotingEntity {
```

Comment:
I was wondering if you had any thoughts regarding the way subscription “methods” would get declared here?

Reply:
My plan was to start experimenting with that at the org/core level, and then mimic whatever solution we find there, over here.